testfixtures.go

```diff
@@ -268,7 +268,7 @@ func fixturesFromFolder(folderName string) ([]*fixtureFile, error) {
 	}
 
 	for _, fileinfo := range fileinfos {
-		if !fileinfo.IsDir() && filepath.Ext(fileinfo.Name()) == ".yml" {
+		if !fileinfo.IsDir() && (filepath.Ext(fileinfo.Name()) == ".yml" || filepath.Ext(fileinfo.Name()) == ".yaml") {
```

Comment:
What about caching the result of `filepath.Ext(fileinfo.Name())` into its own variable?

Reply:
Hej,

you're absolutely right! 🙈